### PR TITLE
distributed version of getLangs

### PR DESCRIPTION
**Pull request by ymatsuda**
@raymondng @eishay 
Search inspects the language distribution of user's keeps for better language detection. This change supports it in a distributed search environment.

### CODE REVIEW

**Comment by raymondng** [comment]
Interesting -- I didn't know that we do this. Nice :+1:

File: kifi-backend/modules/search/app/com/keepit/search/SearchCommander.scala [hunk 1]
```diff
@@ -310,7 +325,14 @@ class SearchCommanderImpl @Inject() (
       }
     }
 
-    val langProf = monitoredAwait.result(langProfFuture, 10 seconds, "slow getting lang profile")
+    val langProf = {
+      val freqs = monitoredAwait.result(Future.sequence(resultFutures), 10 seconds, "slow getting lang profile")
```
Comment (raymondng) [inline kifi-backend/modules/search/app/com/keepit/search/SearchCommander.scala:329]: Interesting. Would be very nice if we can track which partial search result is the long pull.

**Comment by raymondng** [comment]
lgtm. thanks!

**Comment by eishay** [comment]
cool!

**Pull request merged**